shared/chat/conversation/list-area/index.desktop.tsx

```diff
@@ -237,27 +237,39 @@ const useScrolling = (p: {
       })
   }, [listRef, adjustScrollAndIgnoreOnScroll, checkForLoadMoreThrottled])
 
+  const scrollCheckRef = React.useRef<ReturnType<typeof setTimeout>>()
+  React.useEffect(() => {
+    return () => {
+      clearTimeout(scrollCheckRef.current)
+    }
+  }, [])
+
   // While scrolling we disable mouse events to speed things up. We avoid state so we don't re-render while doing this
-  const onScrollThrottled = C.useDebouncedCallback(
+  const onScrollThrottled = C.useThrottledCallback(
     React.useCallback(() => {
-      // starting a scroll
+      clearTimeout(scrollCheckRef.current)
+      scrollCheckRef.current = setTimeout(() => {
+        if (isScrollingRef.current) {
+          isScrollingRef.current = false
+          if (pointerWrapperRef.current) {
+            pointerWrapperRef.current.classList.remove('scroll-ignore-pointer')
+          }
+
+          const list = listRef.current
+          // are we locked on the bottom?
+          if (list) {
+            lockedToBottomRef.current =
+              list.scrollHeight - list.clientHeight - list.scrollTop < listEdgeSlopBottom
+          }
+        }
+      }, 200)
+
       if (!isScrollingRef.current) {
+        // starting a scroll
         isScrollingRef.current = true
         if (pointerWrapperRef.current) {
           pointerWrapperRef.current.classList.add('scroll-ignore-pointer')
         }
-      } else {
```

Comment:
this old way made an assumption that we'd get both calls but this turns out to not be true so we explicitly have to fire another timer